src/components/swap/SwapModalFooter.tsx

```diff
@@ -83,6 +91,7 @@ export default function SwapModalFooter({
             allowedSlippage,
             transactionDeadlineSecondsSinceEpoch,
             isAutoSlippage,
+            /** isAutoRouterApi= */ !clientSideRouter,
```

Comment:
i think you should make the argument to `formatAnalyticsEventProperties` a single object, that way all the parameters are named as keys

Reply:
sgtm done